wagn/lib/wagn/generators/wagn/templates/config/routes.erb

```diff
@@ -1,8 +1,6 @@
 # -*- encoding : utf-8 -*-
 
-require 'decko/engine'
-
-Rails.application.routes.draw do
+Wagn.application.routes.draw do
```

Comment:
this is in Wagn.  Engine decks won't use this.

Please try to avoid using the Rails constant wherever possible.  Bit by bit we may be able to drop our dependencies on rails gems.